### PR TITLE
Fix numba version deprecation warning

### PR DESCRIPTION
**Pull request by levkk**
#### What does this implement/fix? Explain your changes.
Screenshot from 2020-05-16 10-15-33

### CODE REVIEW

**Comment by bmcfee** [comment]
@levkk Thanks for this -- the initial commit was fine, but I'm confused about the reversion to the old string check (with the newer version).

If you roll back the last commit, I think this would be good to go.

Don't worry about the failing test on travis; it's due to a combination of issues that are out of scope for this one, and handled by a separate PR.

File: librosa/util/decorators.py [hunk 1]
```diff
@@ -4,9 +4,10 @@
 '''Helpful tools for deprecation'''
 
 import warnings
+from packaging import version
```
Comment (bmcfee) [inline librosa/util/decorators.py:7]: Can you add `packaging` to the requirements list in setup.py?

Otherwise looks good!

**Comment by bmcfee** [comment]
BTW, I've merged the fix for the travis build.  If you could rebase this PR to the current master, tests should run cleanly again.  Sorry about that!

File: setup.py [hunk 2]
```diff
@@ -38,6 +38,7 @@
     install_requires=[
         'audioread >= 2.0.0',
         'numpy >= 1.15.0',
+        'packaging >= 20.3',
```
Comment (bmcfee) [inline setup.py:41]: Thanks -- Is 20.3 the minimal version required here?  From a skim of the docs, it looks like anything above 17 would do the trick, but I don't have much experience with this package.

Comment (levkk) [reply setup.py:41]: That's what I had in my virtualenv, totally not required I think. I can put 17.

Comment (levkk) [inline setup.py:41]:
```suggestion
        'packaging >= 17',
```

**Comment by bmcfee** [comment]
LGTM. Thanks again!

**Pull request merged**